aerogear/mobile-developer-console
# Fix: Upgrade to react-scripts@3.1.1

## Motivation
https://issues.jboss.org/browse/AGMS-925

## What
Upgrade to react-scripts@3.1.1

## Why
Fix possible security issues

## Verification Steps
1. Remove node_modules
2. Run `npm install`
3. Run `npm run test` & `npm run lint`
4. Manually check site works

## Checklist:


## Progress


## Additional Notes
- you may be required to remove the package-lock.json file if there is issues running the eslint

PS.: Add images and/or .gifs to illustrate what was changed if this pull request modifies the appearance/output of something presented to the users.
